targets/targets.json

```diff
@@ -571,6 +571,20 @@
         "release_versions": ["2", "5"],
         "device_name": "MKW41Z512xxx4"
     },
+    "RO359B": {
```

Comment:
The idea is that if there is a new target based on K24F MCU, how does that one will work? copy paste files + these json lines?

the tree could be:
- MCU_K24F
  - TARGET_RO359B
  - TARGET_NEW_TARGET_THAT_CAN_BE_ADDED
- here are common K24F files like spi implementation etc